<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,5,1,2" columnstretch="4,11,1">
    <item row="1" column="0" rowspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>1667</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="lineEdit_l">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>м</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Длина стержня</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_F_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>м^2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Максимальная площадь F_2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Минимальная площадь F_1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_F_1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>м^2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Частота колебаний</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_omega">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>320</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Гц</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_Material">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>380</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Материал</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_MaterialName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Название материала</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_MaterialName">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>450</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_MaterialDensity">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>480</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Плотность материала</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_MaterialDensity">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>500</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>кг/м^3</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_MaterialYModule">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>530</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Модуль Юнга</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_MaterialYModule">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>550</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Па</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Тип построения</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_TypeOfTask">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Из 1-ой части (переменное сечение)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Из 2-х частей (переменное - постоянное сечение)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Из 2-х частей (постоянное - переменное сечение)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Из 3-х частей</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Масса на конце</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_M">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>кг</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ExampleTask">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Тестовые данные</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_SaveMaterial">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>420</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/rec/img/addMaterial.png</normaloff>:/rec/img/addMaterial.png</iconset>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="widget_Graph" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_InputDot2">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>10</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_InputDot1">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>10</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_AddDot">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>10</y>
         <width>24</width>
         <height>24</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Добавление точки</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/rec/img/addPoint2.png</normaloff>:/rec/img/addPoint2.png</iconset>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_ColorChange">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Move">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Перемещение графика</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resource.qrc">
           <normaloff>:/rec/img/moving2.png</normaloff>:/rec/img/moving2.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Zoom">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Изменение масштаба</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resource.qrc">
           <normaloff>:/rec/img/zoom.png</normaloff>:/rec/img/zoom.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ClearGraph">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Удаление всех графиков</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resource.qrc">
           <normaloff>:/rec/img/trashcan.png</normaloff>:/rec/img/trashcan.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_OpenFile">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Открытие файла</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resource.qrc">
           <normaloff>:/rec/img/openFolder2.png</normaloff>:/rec/img/openFolder2.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_SaveImage">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Сохранение изображения</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resource.qrc">
           <normaloff>:/rec/img/image.png</normaloff>:/rec/img/image.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_SaveGraph">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Сохранение данных</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resource.qrc">
           <normaloff>:/rec/img/save.png</normaloff>:/rec/img/save.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_PlotGraph">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Построение графика</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resource.qrc">
           <normaloff>:/rec/img/addGraph.png</normaloff>:/rec/img/addGraph.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="textBrowser_Out"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_Save">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/rec/img/save_c.png</normaloff>:/rec/img/save_c.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Go">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/rec/img/go2.png</normaloff>:/rec/img/go2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Клик!</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
